packages/wdio-sauce-service/src/index.js

```diff
@@ -151,26 +104,36 @@ export default class SauceService {
             return
         }
 
-        return this.updateJob(this.sessionId, this.failures)
-    }
+        const status = 'status: ' + (this.failures > 0 ? 'failing' : 'passing')
 
-    onReload (oldSessionId, newSessionId) {
-        if (!this.sauceUser || !this.sauceKey) {
-            return
+        if (!this.isMultiremote) {
+            log.info(`Update job with sessionId ${global.browser.sessionId}, ${status}`)
+            return this.updateJob(global.browser.sessionId, this.failures)
         }
 
-        this.sessionId = newSessionId
-        return this.updateJob(oldSessionId, this.failures, true)
+        return Promise.all(Object.entries(this.capabilities).map(([browserName, instance]) => {
+            log.info(`Update multiremote job for browser "${browserName}" and sessionId ${global.browser[browserName].sessionId}, ${status}`)
+            return this.updateJob(global.browser[browserName].sessionId, this.failures, false, browserName)
+        }))
     }
 
-    updateJob (sessionId, failures, calledOnReload = false) {
+    // onReload (oldSessionId, newSessionId) {
```

Comment:
Can be removed?

Reply:
not really, I wanna make sure that this works to.